extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -113,6 +113,8 @@ require([
 			},
 			lang: videoDetails.lang
 		}, onPlayerReady);
+
+		featuredVideoMoatTracking.loadTrackingPlugin();
```

Comment:
Isn't it too late? Player is ready at this point. Can't we load main plugin library earlier?